src/app/lib/utilities/usePrevious/index.test.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import usePrevious from '.';
+
+describe('usePrevious', () => {
+  let value;
+  const initialProp = 'first';
+
+  const ComponentWithHook = ({ prop }) => {
+    value = usePrevious(prop);
+    return null;
+  };
+
+  const wrapper = mount(<ComponentWithHook prop={initialProp} />);
+
+  it('should set initial value to null', () => {
+    expect(value).toBe(null);
+    wrapper.setProps({ prop: 'second' });
+  });
+
+  it('should return previous value after updates to dependency', () => {
```

Comment:
Does `updates to dependency` make sense?

Reply:
@rhell I think `dependency` makes sense to use because it's a hook. does `after dependency has changed` reads better ?